tests/unit/moduleapi/basics.tcl

```diff
@@ -38,4 +38,17 @@ start_server {tags {"modules external:skip"} overrides {enable-module-command no
     test {module command disabled} {
        assert_error "ERR *MODULE command not allowed*" {r module load $testmodule}
     }
+}
+
+set modules [list loadmodule $testmodule]
+start_cluster 3 0 [list config_lines $modules] {
+    set node1 [srv 0 client]
+    set node2 [srv -1 client]
+    set node3 [srv -2 client]
+
+    test "Verify RM_Call inside module load function on cluster mode" {
+        assert_equal {PONG} [$node1 PING]
+        assert_equal {PONG} [$node2 PING]
+        assert_equal {PONG} [$node3 PING]
+    }
```

Comment:
I think this should be in `unit/moduleapi/cluster.tcl`?

Reply:
Moved